package.json

```diff
@@ -20,7 +20,7 @@
   "dependencies": {
     "@openeth/truffle-typings": "0.0.6",
     "@openzeppelin/contracts": "~3.2.0",
-    "@rsksmart/rif-relay-contracts": "https://github.com/anarancio/rif-relay-contracts#PP-94/revenue-sharing-model",
+    "@rsksmart/rif-relay-contracts": "https://github.com/antomor/rif-relay-contracts#PP-94/fix-truffle-typings-patch",
```

Comment:
We need to revert this change after merging the [rif-relay-contracts PR](https://github.com/anarancio/rif-relay-contracts/pull/21)